### ISSUE TITLE
Remove bottom fade overlay in main content for database layouts

### ISSUE THREAD

**Issue by LibbyUX** [cns-iu/cns-website#124]
# Summary
The bottom fade overlay lags a bit on mobile devices, so it falls out of sync with the page scroll. I don't think it's necessary for the UX, as main content usually doesn't have a fade, and we see this pattern in other UIs more commonly in smaller components.

## Todo
- [ ] Remove bottom fade overlay from desktop and mobile layouts:
  - [ ] Research database layout
  - [ ] People database layout

### People

https://github.com/user-attachments/assets/abcebd82-558a-4b30-9ff8-91b9de3e28c4

### Research

https://github.com/user-attachments/assets/04a4f40b-314e-4001-a71e-1fc771c8aa68
